### PR TITLE
chore: better typing for document

### PR DESCRIPTION
**Pull request by Nebulis**
- created top-level OpenAttestation type that allow both v2 and v3 document
- data in WrappedDocument changed from any to OpenAttestation

### CODE REVIEW

File: src/@types/document.ts [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import { OpenAttestationDocument as OpenAttestationDocumentV2 } from "../__generated__/schemaV2";
```
Comment (yehjxraymond) [inline src/@types/document.ts:1]: Would you like to add a comment that it's generated by quicktype and can see in postinstall? I was wondering where this came from

File: src/schema/2.0/schema.test.ts [hunk 2]
```diff
@@ -298,7 +303,7 @@ describe("schema/v2.0", () => {
             {
               name: "ABC",
               identityProof: {
-                type: "DNS-TXT",
+                type: IdentityProofType.DNSTxt,
```
Comment (yehjxraymond) [inline src/schema/2.0/schema.test.ts:306]: Why did you change some of these and not those on top? I'll honestly prefer to keep them as "DNS-TXT" in the test to show that the type didnt change after this PR though

File: src/@types/document.ts (hunk 1 above)
Comment (Nebulis) [reply src/@types/document.ts:1]: done =)

File: src/schema/2.0/schema.test.ts (hunk 2 above)
Comment (Nebulis) [reply src/schema/2.0/schema.test.ts:306]: will see in another MR

**Pull request merged**